src/ui/hooks/useGameRules.ts

```diff
@@ -58,10 +38,8 @@ export const useGameRules = () => {
   assertStringIsGameState(resolvedGameState)
 
   return {
-    eventCardsThatCanBePlayed,
     game,
     gameState: resolvedGameState,
-    selectedWaterCardInHandIdx,
-    winner,
+    winner: game.winner,
```

Comment:
It seems that `winner` should only be within `game` here, so this line can probably be removed.